src/lib/uri/uri.h

```diff
@@ -13,6 +13,52 @@
 extern "C" {
 #endif /* defined(__cplusplus) */
 
+struct uri_option_value {
+        /** Each URI option value can be represented by string. */
+        char *value;
+};
+
+struct uri_option_values {
+        /**
+         * Copies of strings, which contains one or several option values.
+         * We should work with copies of appropriate string, because lua
+         * strings are constants.
+         */
+        char **copies;
+        /** Count of `copies` array. */
+        int copies_count;
+        /** Array of URI option values. */
+        struct uri_option_value *values;
+        /** Count of URI option values. */
+        int values_count;
```

Comment:
Using plural before 'count' is grammatically incorrect. You should use `value_count`, `copy_count`, etc.